submission/submission-webapp/src/main/resources/application.yml

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Why did you add these two lines? Is it a style guideline?

Reply:
mistake